CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing
+
+Contributions are **welcome** and will be fully **credited**.
+
+We accept contributions via Pull Requests on [Github](https://github.com/phpflo/phpflo).
+
+
+## Pull Requests
```

Comment:
This would make sense in a [PR and issue template](https://github.com/blog/2111-issue-and-pull-request-templates) [see inferno for an example](https://github.com/infernojs/inferno/tree/master/.github)